### DISCUSSION TITLE
Can't connect 'Scarlett Solo Studio' Audio Interface to Jamulus in Linux Ubuntu?

### DISCUSSION THREAD

**Discussion by ThulithaNawagamuwa** [jamulussoftware/jamulus#2367] [Technical Support & Help]
I can connect the "Scarlett Solo Studio" to Ubuntu but the problem is Jamulus won't detect it. I have selected the input and output audio devices as Scarlett Solo in the advanced settings of Jack Audio Connection kit (QjackCtl  -  version is 0.4.5). 

I can play both record and play an audio in ubuntu using Scarlett Solo but **with Jamulus I can't connect audio using Scarlett Solo.** However audio can be connected to Jamulus using my inbuilt soundcard driver. 

Can I use Scarlett solo driver in Ubuntu? And how to connect an audio through scarlett audio interface to Jamulus in Ubuntu OS?

I really appreciate any help.

Thank you.

**Comment by ignotus666** [comment]
Have you connected the Scarlett device to Jamulus in QjackCtl's "Connect" (or "Graph") window?

**Reply by ThulithaNawagamuwa** [reply]
No.  I haven't tried that.

**Reply by jujudusud** [reply]
https://jamulus.io/wiki/Installation-for-Linux

**Reply by ThulithaNawagamuwa** [reply]
I have done this method. But Scarlett Solo model won't be detected by Jamulus

**Reply by jujudusud** [reply]
You have to select JACK in Jamulus.

**Reply by ignotus666** [reply]
With QjackCtl launched Jamulus will only give you the option of selecting JACK - and that's fine. In the "Connect" window of QjackCtl, you have to connect the Scarlett outputs on the left to the Jamulus inputs on the right. If it's not already, you also have to connect the outputs of Jamulus (left) to the Scarlett inputs (right).

**Reply by ann0see** [reply]
You need to select it in the "Parameter" not advanced tab.

**Reply by jujudusud** [reply]
@ThulithaNawagamuwa, is it ok now ?
